app/src/main/java/mozilla/org/webmaker/activity/Project.java

```diff
@@ -7,4 +9,16 @@
     public Project() {
         super("project", R.id.project_layout, R.layout.project_layout, R.menu.menu_project);
     }
+
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
```

Comment:
We should be ok here. Most of my changes to the `onCreateOptionsMenu` landed in #1658